### PR TITLE
Added readme configurations files and links

### PR DESCRIPTION
**Pull request by rasanjaya85**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: azure-simple/packer/ansible-is/dev/inventory [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[is]
+is_1 ansible_host=localhost ansible_user=centos ansible_connection=local
+
```
Comment (chirangaalwis) [inline azure-simple/packer/ansible-is/dev/inventory:3]: @rasanjaya85 shall we remove this unnecessary EOF line?

File: azure-simple/terraform/cloudinit/compute-template.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#cloud-config
+
+mounts:
+ - [ "//isstorageshare.file.core.windows.net/isshare", /mnt/sharedfs, "cifs", "vers=3.0,username=isstorageshare,password=${storage_access_key},dir_mode=0777,file_mode=0777,serverino", "0", "0"]
+
+runcmd:
+ - sed -i 's|CONNECTION_STRING|${db_connection_strings}|g' /tmp/ansible-is/dev/group_vars/is.yml
+ - cd /tmp/ansible-is && ansible-playbook -i dev/inventory site.yml
+
```
Comment (chirangaalwis) [inline azure-simple/terraform/cloudinit/compute-template.yaml.tpl:9]: @rasanjaya85 shall we please remove this unnecessary EOF line? Please check other files also for such additional lines.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: azure-simple/packer/ansible-is/dev/inventory (hunk 1 above)
Comment (rasanjaya85) [reply azure-simple/packer/ansible-is/dev/inventory]: Done

File: azure-simple/terraform/cloudinit/compute-template.yaml.tpl (hunk 2 above)
Comment (rasanjaya85) [reply azure-simple/terraform/cloudinit/compute-template.yaml.tpl]: Done

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**